yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -137,10 +140,19 @@ func InitAssessmentDB() error {
 			parent_table_name   TEXT,
 			size_in_bytes       INTEGER,
 			PRIMARY KEY(schema_name, object_name));`, TABLE_INDEX_STATS),
-		// to store pgss output for unsupported query constructs detection
+		// to store pgss output for performance validation and unsupported query constructs detection
+		// Schema exactly matches src/pgss.QueryStats struct for consistency
 		fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
-			queryid			BIGINT,
-			query		TEXT);`, DB_QUERIES_SUMMARY),
+			queryid				BIGINT,
+			query				TEXT,
+			calls				BIGINT,
+			rows				BIGINT,
+			total_exec_time		REAL,
+			mean_exec_time		REAL,
+			min_exec_time		REAL,
+			max_exec_time		REAL,
+			stddev_exec_time	REAL,
+			PRIMARY KEY (queryid));`, DB_QUERIES_SUMMARY),
```

Comment:
Thoughts on changing the name?
`DB_QUERIES_SUMMARY` --> `DB_QUERY_STATS`?

Reply:
Just thinking if it can be a breaking change.
Export Schema only uses assessment report struct, so it should not be right? @makalaaneesh